Independer.WCFDataAnnotations/ValidatingParameterInspector.cs

```diff
@@ -70,9 +71,12 @@ public class ValidatingParameterInspector : IParameterInspector {
       var validationResults = new List<ValidationResult>();
 
       foreach (var input in inputs) {
-        foreach (var validator in GetValidators(parameterPosition)) {
+        var parameterDetails = GetParameterDetails(parameterPosition);
+
+        foreach (var validator in GetValidators(parameterDetails)) {
           var results = validator.Validate(input);
-          validationResults.AddRange(results);
+          
+          validationResults.AddRange(AddParameterNamesWhenMissing(results, parameterDetails?.Name));

```

Comment:
not doing anything with operationName there?

Reply:
Nope, few lines below, the operation name is added to the Fault message and to the logging already. (you should check the view 'whole file', because it seems to hide some parts in the default view)